src/WebJobs.Script/Metrics/HostMetrics.cs

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Diagnostics.Metrics;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Azure.WebJobs.Script.Metrics
+{
+    public class HostMetrics
+    {
+        /// <summary>
+        /// Gets or sets the function group name. This can be either http, durable or
+        /// the name of a function.
+        /// </summary>
+        public string FunctionGroup { get; set; }
+
+        /// <summary>
+        /// Gets or sets the total number of permanent host failures.
+        /// </summary>
+        public long AppFailureCount { get; set; }
+
+        /// <summary>
+        /// Gets or sets the total number of in-progress function invocations.
+        /// </summary>
+        public long ActiveInvocationCount { get; set; }
+
+        /// <summary>
+        /// Gets or sets the total number of function invocations that have started.
+        /// </summary>
+        public long StartedInvocationCount { get; set; }
```

Comment:
The existing ExecutionCount metric we're already logging is a count of the completed invocations during an interval. Wondering if we need this separate count.

Reply:
Bala would like to capture all executions even if they don't complete so I thought it better to create a new metric for it. I believe this is to be a running total as well